package.json

```diff
@@ -166,6 +165,8 @@
     "chai-spies": "^1.0.0",
     "depcheck": "^1.4.2",
     "eslint-plugin-no-only-tests": "^2.6.0",
+    "msw": "^0.35.0",
```

Comment:
need 0.35+ for `msw-storybook-addon`, the latest msw broke all of the old stories so I just kicked it up 1 version